scene/main/scene_tree.cpp

```diff
@@ -1135,11 +1135,11 @@ Error SceneTree::change_scene(const String &p_path) {
 }
 
 Error SceneTree::change_scene_to(const Ref<PackedScene> &p_scene) {
+	ERR_FAIL_NULL_V(p_scene, ERR_CANT_CREATE);
+
 	Node *new_scene = nullptr;
```

Comment:
```suggestion
	Node *new_scene = p_scene->instantiate();
```

Reply:
This is reasonable - I didn't really take into consideration that the new flow wouldn't require that to be split up into two steps, haha.